jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/RabitTracker.java

```diff
@@ -55,11 +55,15 @@ public void run() {
       String line;
       try {
         while ((line = reader.readLine()) != null) {
-          trackerProcessLogger.info(line);
+          trackerProcessLogger.error(line);
```

Comment:
Could you please remove this error since we have the exit code check now?

Reply:
return value is not enough for user to find root cause, maybe stderr is more helpful.  what do you think about?

Reply:
There is none code printing something to stderr in tracker.py, so it should have some error during executing tracker.py, I think it's reasonable to use error level during loging strerr message.

Reply:
If this error is kept then the exit code validation is dead code and will never be run. We have to pick one of them.

Reply:
`trackerProcessLogger.error(line) just logging by ERROR level, it does not affect the rest logic of code`

Reply:
My mistake. Look good  to me then.